sdk/eventgrid/Microsoft.Azure.EventGrid/src/Microsoft.Azure.EventGrid.csproj

```diff
@@ -1,21 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-	<!-- Please do not move/edit code below this line -->
-
 	<PropertyGroup>
-		<PackageId>Microsoft.Azure.EventGrid</PackageId>
+	    <AssemblyTitle>Microsoft Azure EventGrid Library</AssemblyTitle>
 		<Description>This library can be used to publish events to Azure Event Grid and to consume events delivered by EventGrid. It also defines the event schemas for the events published to EventGrid by various Azure services. </Description>
-		<Version>3.1.0</Version>
-		<AssemblyName>Microsoft.Azure.EventGrid</AssemblyName>
+		<VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
There looks to be a difference in versioning. The Version property was 3.1.0 in the project but 3.0.0 in the assembly info file so we should figure out what the correct version should be. I suspect it would need to b 3.1.0.

Reply:
The assembly dll in master are 3.0.0 though.
![Assembly Version](https://user-images.githubusercontent.com/31145988/57033552-0a4c0780-6c02-11e9-9a02-b35d01884fed.PNG)

Reply:
When the package version was updated to 3.1.0 (https://github.com/Azure/azure-sdk-for-net/pull/5145) I suspect they just missed assembly version update. I would just set the VersionPrefix to 3.1.0

Reply:
Ok